backend/python/app/graphql/types/comment_type.py

```diff
@@ -7,6 +7,12 @@ class CreateCommentDTO(graphene.InputObjectType):
     content = graphene.String(required=True)
 
 
+class UpdateCommentDTO(graphene.InputObjectType):
+    id = graphene.Int(required=True)
+    resolved = graphene.Boolean(required=True)
```

Comment:
I dont think lines 12 and 13 are required - users will often only update one or the other